src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -86,7 +86,7 @@ const ProfilePicture = ({
                   style={styles.profileImage(size, width)}
                   source={{
                     uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH,
-                    cache: 'cacheOnly'
+                    cache: 'web'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same as above, consider the frequency of profile picture updates before deciding on the caching strategy.
```diff
- cache: 'web'
+ cache: 'cacheOnly'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->